README.md

```diff
@@ -232,15 +235,46 @@ spotify_user = RSpotify::User.new(hash)
 spotify_user.create_playlist!('my_awesome_playlist') # automatically refreshes token
 ```
 
-## Getting raw response
-
-To get the raw response from Spotify API requests, just toggle the `raw_response` variable:
 
 ```ruby
 RSpotify.raw_response = true
 RSpotify::Artist.search('Cher') #=> (String with raw json response)
 ```
 
+Alternately, you can store a proc that is invoked when a new access token is generated. This give you the
```

Comment:
PR looks great, thx! Some notes:

• I think this should be `Additionally, you can use a proc...`, instead of `Alternately, you can store a proc...`. Since it's not actually an alternative to refresh the token, but a way for you to do something with the access token and its lifetime additionally if you want. The PR also gives the idea this is a new way to refresh the token.

• Looks like you mixed 2 sections of the README, the refreshing token part and the raw response one.